internal/generate/crd/crd.go

```diff
@@ -371,6 +374,26 @@ func setCRDStorageVersion(crd *apiextv1beta1.CustomResourceDefinition) {
 	crd.Spec.Versions[0].Storage = true
 }
 
+func setSDKstamps(crd *apiextv1beta1.CustomResourceDefinition, operatorType string) {
+	annotations := crd.GetAnnotations()
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
```

Comment:
```suggestion
	if len(crd.GetAnnotations()) < 1 {
		annotations = make(map[string]string)
	}
```

Reply:
The only situation in which we need to create a map is if `crd.GetAnnotations() == nil`. Otherwise a non-nil map exists and can be accessed.